eng/Publishing.props

```diff
@@ -47,6 +47,7 @@
     <SdkAssetsToPublish Include="$(ArtifactsShippingPackagesDir)*.svg" Condition=" '$(PublishBinariesAndBadge)' == 'true' ">
       <ExcludeFromManifest>true</ExcludeFromManifest>
     </SdkAssetsToPublish>
+    <SdkNonShippingAssetsToPublish Include="$(ArtifactsNonShippingPackagesDir)*.nupkg" />
```

Comment:
nit: According to @jcagme, the "ExcludeFromManifest" metadata is no longer required for the publish items above, but you can leave this if you're worried about messing around with it.  If you were to remove it and it wasn't supported, you'd see a build break in your internal build validation.